.env

```diff
@@ -1,3 +1,4 @@
 REACT_APP_API_GATEWAY=api/gateway
 REACT_APP_WS_GATEWAY=ws/gateway
+REACT_APP_API_VERSION=v1
```

Comment:
API_VERSION is not global but per service
Currently it's the same number but not later